### ISSUE TITLE
add list attribute to attributes table

### ISSUE THREAD

**Issue by scottaohara** [w3c/html-aria#277]
rules [relating to `input` with `list` attribute and `aria-haspopup`](https://w3c.github.io/html-aria/#el-input-text-list) should be added to the attributes table.

**Comment by scottaohara** [comment]
This update is editorial.  Match the way `input type=checkbox` links to the `aria-checked / checked` rule.
